src/pages/Home/Home.layout.jsx

```diff
@@ -16,13 +17,20 @@ import { Button, Header, Logo, ManImage, Text } from './Home.styled'
  */
 const Home = ({ title, subtitle, buttonText }) => (
   <Header>
-    <Logo src={logo} alt="logo" />
-    <Text as="h1" color="secondary" large>
-      {title}
-    </Text>
-    <Text small>{subtitle}</Text>
-    <ManImage src={deliveryMan} alt="Delivery man" />
-    <Button to="/check">{buttonText}</Button>
+    <HalfSection backgroundColor={Colors.yellow06}>
+      <Logo src={logo} alt="logo" />
+      <ManImage style={{ display: 'nones' }} src={deliveryMan} alt="Delivery man" />
```

Comment:
`display: 'nones'` shouldn't be valid, maybe you meant `display: 'none;'`, alsy why you wanted to hide this?

Reply:
ready